lib/Basics/Utf8Helper.h

```diff
@@ -44,7 +44,7 @@ class RegexMatcher;
 namespace arangodb {
 namespace basics {
 
-enum class LanguageType { INVALID, DEFAULT, ICU };
+enum class LanguageType { INVALID, DEFAULT, ICU, EMPTY };
```

Comment:
I can' t see the change "promised" in the description of the PR about `ULOC_ACTUAL_LOCALE`, is this reverted? Is it needed?

Reply:
Yes, it was reverted after discussion here: https://github.com/arangodb/arangodb/pull/21050#discussion_r1639777585